test/tests.js

```diff
@@ -57,6 +58,112 @@ describe('Dylan', function() {
     });
   });
 
+  it('returns 404s for unhandled requests', (done) => {
+    app.listen(8888);
+
+    chai.request(host)
```

Comment:
Since chai.request(host) is used everywhere you should be able to do something like this throughout tests:

`const request = chai.request(host);`